docs/fast_data/runtime_management/overview.mdx

```diff
@@ -44,35 +44,68 @@ This Fast Data runtime management flexibility, enables strategic management of t
   - Gracefully handle unexpected situations or system maintenance by pausing specific data pipeline steps.
   - Resume operations post-maintenance to minimize downtime and enhance overall system reliability.
 
-### Marketplace Applications
+### Application Deployment
 
-To setup Control Plane, you can start from the [application marketplace](/marketplace/applications/mia_applications.md) and search for `control plane` in the search bar. You will have two options:
+#### Control Plane
 
-- _Fast Data Control Plane_, to link a basic control plane instance to a new or existing Envoy API Gateway. 
-- _Fast Data Control Plane With Access Control_, to link a control plane with an authorization layer (see box tip below for more information)
+The Fast Data Runtime Management solution is distributed as a [Mia-Platform Marketplace application](/marketplace/applications/mia_applications.md), which automatically
+instantiates most of the needed resources and pre-populate configuration files with default values that are quick to edit.
 
-![Control Plane Marketplace](img/marketplace_runtime_management_app_selection.png)
+We strongly recommend to configure such application into a **dedicated Console Project** (for example one named _Data Fabric_) that can be reached from all the other ones.
+In this manner the Fast Data Runtime Management software can run mostly independently of the actual Fast Data runtimes, so that updates to them still allow to receive and visualize
+the Fast Data states from all the others runtimes.
+
+This application can be found in the Marketplace by looking it up in using the keyword `control plane`, which will present you with two options for instantiating its configuration:
+
+- _Fast Data Control Plane_, which will provide the basic Control Plane instance alongside an Envoy API Gateway.
+- _Fast Data Control Plane With Access Control_, which will provide all the configurations of _Fast Data Control Plane_ with the addition of the resources needed for introducing an authorization layer (see box tip below for more information)
```

Comment:
This is no longer true as we updated only the one with access control. We should explain to the user what to do to remove access control in case they don't want it, but we can do this is the application's doc